docs/integrations-in-rancher/cluster-api/cluster-api.md

```diff
@@ -9,6 +9,6 @@ title: Cluster API (CAPI) with Rancher Turtles
 [Rancher Turtles](https://turtles.docs.rancher.com/) is a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/#operators-in-kubernetes) that manages the lifecycle of provisioned Kubernetes clusters, by providing integration between your Cluster API (CAPI) and Rancher. With Rancher Turtles, you can:
 
 - Import CAPI clusters into Rancher, by installing the Rancher Cluster Agent in CAPI provisioned clusters.
-- Configure the [CAPI Operator](https://turtles.docs.rancher.com/turtles/next/en/reference-guides/rancher-turtles-chart/values.html#cluster-api-operator-values).
+- Configure the [CAPI Operator](https://turtles.docs.rancher.com/turtles/next/en/operator/chart.html#_cluster_api_operator_values).
```

Comment:
```suggestion
- Configure the [CAPI Operator](https://turtles.docs.rancher.com/turtles/v0.18/en/operator/chart.html#_cluster_api_operator_values).
```

/docs is a mirror of the most recent released version (e.g. version-2.11 as of today) so it should match the linking to v0.18.